iOS Client/PodcastsController/PodcastDetailViewController.swift

```diff
@@ -71,6 +90,25 @@ class PodcastDetailViewController: UIViewController, StoreSubscriber {
         }
     }
     
+    func animateImage(size: CGSize, duration: CFTimeInterval) {
+        var shadowRect = imageView.bounds
+        var fromRect = CGRect(x: shadowRect.origin.x, y: shadowRect.origin.y, width: shadowRect.width + 10, height: shadowRect.height + 10)
+        shadowRect.size = CGSize(width: size.width + 10, height: size.height + 10)
+        let shadowAnimation = CABasicAnimation(keyPath: "shadowPath")
+        shadowAnimation.fromValue = UIBezierPath(rect: fromRect).cgPath
+        shadowAnimation.toValue = UIBezierPath(rect: shadowRect).cgPath
+        shadowAnimation.isRemovedOnCompletion = false
+        shadowAnimation.fillMode = kCAFillModeForwards
+        shadowAnimation.duration = duration
+        shadowAnimation.timingFunction = CAMediaTimingFunction(name: kCAMediaTimingFunctionEaseInEaseOut)
+        self.imageWidthConstraint.constant = size.width
+        self.imageHeightConstraint.constant = size.height
```

Comment:
strong, it's always a good day when you get to reference auto layout constraints in your code